integrations/docker/images/chip-build-k32w/Dockerfile

```diff
@@ -0,0 +1,16 @@
+ARG VERSION=latest
+FROM connectedhomeip/chip-build:${VERSION}
+
+# Setup the K32W SDK
+RUN set -x \
+    && apt-get update \
+    && DEBIAN_FRONTEND=noninteractive apt-get install -y wget unzip \
+    && mkdir -p /opt/sdk \
+    && cd /opt/sdk \
+    && wget https://mcuxpresso.nxp.com/eclipse/sdk/2.6.2/plugins/com.nxp.mcuxpresso.sdk.sdk_2.x_k32w061dk6_2.6.2.201911251446.jar \
+    && unzip com.nxp.mcuxpresso.sdk.sdk_2.x_k32w061dk6_2.6.2.201911251446.jar \
```

Comment:
Would it make sense to make this part of the docker image?  If so, a followup for that would be great.

Reply:
How can I do that? Isn't already of the Docker image if it's included in the Dockerfile?